src/app/cards/munchkin/cards/treasures/permanent-item/edit/permanent-item-edit.component.html

```diff
@@ -0,0 +1,15 @@
+<div>
+  <app-editable-string [(string)]="card.name" [label]="'name'"></app-editable-string>
+
+  <app-editable-string [(string)]="card.bonus" [label]="'bonus'">
+  </app-editable-string>
+  <app-editable-string [(string)]="card.specificity" [label]="'specificity'">
+  </app-editable-string>
+  <app-editable-rich-text [(text)]="card.description" [label]="'description'" [multiline]="'true'">
+  </app-editable-rich-text>
+  <app-editable-string [(string)]="card.kind" [label]="'kind'">
+  </app-editable-string>
+  <input type="checkbox" [(ngModel)]="card.isHuge" [value]="card.isHuge"> Huge
+  <app-editable-number [(number)]="card.price" min="0" max="2000" step="100" [label]="'price'">
```

Comment:
How about having `min` and `max` set in the environment or per-implementation basis?
Also, `step` could be moved out to a component constant.

Reply:
Good point, created ticket for that